main.nf

```diff
@@ -1,352 +1,259 @@
- nf-core/bacterialmappingphylogeny Analysis Pipeline.
+ nf-core/bactmap Analysis Pipeline.
  #### Homepage / Documentation
- https://github.com/nf-core/bacterialmappingphylogeny
+ https://github.com/nf-core/bactmap
 ----------------------------------------------------------------------------------------
 */
 
+nextflow.enable.dsl = 2
 
-def helpMessage() {
-    // TODO nf-core: Add to this help message with new command line parameters
-    log.info"""
-    =======================================================
-                                              ,--./,-.
-              ___     __   __   __   ___     /,-._.--~\'
-        |\\ | |__  __ /  ` /  \\ |__) |__         }  {
-        | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
-                                              `._,._,\'
+////////////////////////////////////////////////////
+/* --               PRINT HELP                 -- */
+////////////////////////////////////////////////////
 
-     nf-core/bacterialmappingphylogeny v${workflow.manifest.version}
-    =======================================================
-
-    Usage:
-
-    The typical command for running the pipeline is as follows:
-
-    nextflow run nf-core/bacterialmappingphylogeny --reads '*_R{1,2}.fastq.gz' -profile standard,docker
-
-    Mandatory arguments:
-      --reads                       Path to input data (must be surrounded with quotes)
-      --genome                      Name of iGenomes reference
-      -profile                      Configuration profile to use. Can use multiple (comma separated)
-                                    Available: standard, conda, docker, singularity, awsbatch, test
-
-    Options:
-      --singleEnd                   Specifies that the input is single end reads
-
-    References                      If not specified in the configuration file or you wish to overwrite any of the references.
-      --fasta                       Path to Fasta reference
-
-    Other options:
-      --outdir                      The output directory where the results will be saved
-      --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
-
-    AWSBatch options:
-      --awsqueue                    The AWSBatch JobQueue that needs to be set when running on AWSBatch
-      --awsregion                   The AWS Region for your AWS Batch job to run on
-    """.stripIndent()
-}
-
-/*
- * SET UP CONFIGURATION VARIABLES
- */
-
-// Show help emssage
-if (params.help){
-    helpMessage()
+def json_schema = "$projectDir/nextflow_schema.json"
+if (params.help) {
+    def command = "nextflow run nf-core/bactmap --input samplesheet.csv --reference ref.fasta -profile docker"
+    log.info Schema.params_help(workflow, params, json_schema, command)
```

Comment:
```suggestion
    log.info NfcoreSchema.params_help(workflow, params, json_schema, command)
```

Reply:
`Schema.groovy` under the `lib/` folder has been renamed to `NfcoreSchema`.  So unless you added custom functions to `Schema.groovy`, which shouldn't be the case, you can safely remove the `Schema.groovy`.